crates/tabby/src/serve/health.rs

```diff
@@ -1,22 +1,39 @@
-use std::sync::Arc;
+use std::{env::consts::ARCH, sync::Arc};
 
 use axum::{extract::State, Json};
 use serde::{Deserialize, Serialize};
+use sysinfo::{CpuExt, System, SystemExt};
 use utoipa::ToSchema;
 
 #[derive(Serialize, Deserialize, ToSchema, Clone, Debug)]
 pub struct HealthState {
     model: String,
     device: String,
     compute_type: String,
+    arch: String,
+    cpu_info: String,
+    cpu_count: u32,
```

Comment:
would change type to `usize` avoid casting in line 36? if so, please change the type